src/bdk.udl

```diff
@@ -129,15 +136,14 @@ interface Wallet {
   sequence<Transaction> get_transactions();
   Network get_network();
   [Throws=BdkError]
-  void sync(BdkProgress progress_update, u32? max_address_param);
-  [Throws=BdkError]
-  string broadcast([ByRef] PartiallySignedBitcoinTransaction psbt);
+  void sync([ByRef] Blockchain blockchain, Progress? progress);
 };
 
 interface PartiallySignedBitcoinTransaction {
   [Throws=BdkError]
   constructor(string psbt_base64);
   string serialize();
+  string txid();
```

Comment:
Just a question: doesn't the txid change over the course of the lifetime of the psbt if the inputs are not using SegWit? Could that be a problem here?